core/src/components/datetime/datetime.tsx

```diff
@@ -74,6 +74,10 @@ import {
  * @slot title - The title of the datetime.
  * @slot buttons - The buttons in the datetime.
  * @slot time-label - The label for the time selector in the datetime.
+ * 
+ * @part picker-item - The individual items when using a wheel style layout, or in the
+ * month/year picker when using a grid style layout.
+ * @part picker-item active - The currently selected picker-item.
```

Comment:
I couldn't just do `@part active` here since that part may apply to other elements too (such as selected dates) so I went with this since it's how the code will look (e.g. `::part(picker-item active)`).

I'm open to changing the active part name to `picker-item-active` instead. The [W3C guidelines](https://drafts.csswg.org/css-shadow-parts/#example-72ff019b) imply that state-based shadow parts should be non-specific (i.e. it should be `part="picker-item active"` and not `part="picker-item picker-item-active"`) but doing it that way does lead to somewhat awkward documentation.